backend/store/error_groups.go

```diff
@@ -229,28 +227,19 @@ func (store *Store) updateErrorGroupState(ctx context.Context,
 	err = store.CreateErrorGroupActivityLog(ctx, model.ErrorGroupActivityLog{
 		Admin:        admin,
 		EventType:    eventType,
-		ErrorGroupID: errorGroup.ID,
+		ErrorGroupID: params.ID,
 		EventData:    eventData,
 	})
 
 	if err != nil {
-		return errorGroup, err
-	}
-
-	// For user-driven state updates, write the error group directly to Clickhouse.
-	// Write to the data sync queue as well to guarantee eventual consistency.
-	if admin != nil {
-		err = store.clickhouseClient.WriteErrorGroups(ctx, []*model.ErrorGroup{&errorGroup})
```

Comment:
Are we good to remove this? (c.c. @Vadman97 @mayberryzane) This creates a small lag in which if the user manually resolves an error, they may still see it in the search until the datasync queue catches up

Reply:
@SpennyNDaJets do you know what kind of delay we see locally?

Reply:
Its a little awkward, so thinking of adding this back. Since the page reloads, this error typically sits at the top with the "Resolved" tag.

Was also looking into sorting the errors by the last instance timestamp so this will also be an improvement, but part of a different PR

Reply:
sounds good, i guess we should keep this then and just make sure we query the error group from postgres here?

Reply:
Moved into the `UpdateErrorGroupStateByAdmin`